### PR TITLE
Fix/android/web view

### LINKED ISSUE #84: [Android] Close WebView after authentication
As a user,
I need the WebView to close after I authenticate my account

What it is now

The user have to close the WebView manually

What it should be

After the authentication the WebView closes automatically

Acceptance criteria
After the authentication the WebView closes automatically

### LINKED ISSUE #244: [Android] Fix Accounts SMS

### PR DESCRIPTION
**Pull request by ricardobrg**
# Issues Addressed
resolves #84
resolves #244

## Additional Information
During tests it did not require 2FA after first verification. We need to test further to check if it was solved in the SDK level. 
Possible fix for #244

### CODE REVIEW

**Comment by ricardobrg** [comment]
resolves #244 
Checking "remember this device" box doesn't asks for OTP in SMS.

File: android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/ReceiptCapture.kt [hunk 1]
```diff
@@ -33,17 +33,24 @@ class ReceiptCapture {
      * @param call The plugin call object.
      * @param activity The Android application activity.
      */
-    fun initialize(call: PluginCall, activity: Context) {
+    fun initialize(call: PluginCall, activity: AppCompatActivity, initializeCallback: () -> Unit) {
```
Comment (Schuler-Gabriel) [inline android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/ReceiptCapture.kt:36]: remove initializeCallback

File: android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/ReceiptCapture.kt [hunk 2]
```diff
@@ -33,17 +33,24 @@ class ReceiptCapture {
...
             physical.initialize(req, activity) { msg, data -> call.reject(msg, data) }.await()
             email.initialize(req, activity) { msg, data -> call.reject(msg, data) }.await()
             retailer.initialize(req, activity) { msg, data -> call.reject(msg, data) }.await()
+            initializeCallback()
             val rsp = RspInitialized(true)
             call.resolve(JSObject.fromJSONObject(rsp.toJson()))
         }
     }
 
+    fun scanInitialize( call: PluginCall, activity: AppCompatActivity){
```
Comment (Schuler-Gabriel) [inline android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/ReceiptCapture.kt:48]: remove scanInitialize

File: android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/ReceiptCapturePlugin.kt [hunk 3]
```diff
@@ -48,7 +48,9 @@ class ReceiptCapturePlugin : Plugin() {
      * @param call The Capacitor plugin call instance.
      */
     @PluginMethod
-    fun initialize(call: PluginCall) = receiptCapture.initialize(call, activity)
+    fun initialize(call: PluginCall) = receiptCapture.initialize(call, activity){
```
Comment (Schuler-Gabriel) [inline android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/ReceiptCapturePlugin.kt:51]: remove the closure

**Review by Schuler-Gabriel — APPROVED** [review]

**Pull request merged**